### ISSUE TITLE
ArrayIndexOutOfBounds when breaking block

### ISSUE THREAD

**Issue by Archy-X** [Minestom/Minestom#364]
Running version b9864a1a28 gives this error when breaking a block in an instance:

```
java.lang.ArrayIndexOutOfBoundsException: Index -21 out of bounds for length 512
	at Minestom Root ClassLoader//net.minestom.server.instance.palette.Section.getBlockAt(Section.java:124)
	at Minestom Root ClassLoader//net.minestom.server.instance.palette.PaletteStorage.getBlockAt(PaletteStorage.java:64)
	at Minestom Root ClassLoader//net.minestom.server.instance.DynamicChunk.getBlockAt(DynamicChunk.java:434)
	at Minestom Root ClassLoader//net.minestom.server.instance.DynamicChunk.getBlockStateId(DynamicChunk.java:164)
	at Minestom Root ClassLoader//net.minestom.server.instance.InstanceContainer.executeNeighboursBlockPlacementRule(InstanceContainer.java:330)
	at Minestom Root ClassLoader//net.minestom.server.instance.InstanceContainer.UNSAFE_setBlock(InstanceContainer.java:214)
	at Minestom Root ClassLoader//net.minestom.server.instance.InstanceContainer.setBlock(InstanceContainer.java:142)
	at Minestom Root ClassLoader//net.minestom.server.instance.InstanceContainer.setSeparateBlocks(InstanceContainer.java:122)
	at Minestom Root ClassLoader//net.minestom.server.instance.BlockModifier.setSeparateBlocks(BlockModifier.java:100)
	at Minestom Root ClassLoader//net.minestom.server.instance.InstanceContainer.breakBlock(InstanceContainer.java:394)
	at Minestom Root ClassLoader//net.minestom.server.listener.PlayerDiggingListener.breakBlock(PlayerDiggingListener.java:183)
	at Minestom Root ClassLoader//net.minestom.server.listener.PlayerDiggingListener.playerDiggingListener(PlayerDiggingListener.java:65)
	at Minestom Root ClassLoader//net.minestom.server.listener.manager.PacketListenerManager.processClientPacket(PacketListenerManager.java:100)
	at Minestom Root ClassLoader//net.minestom.server.network.packet.client.ClientPlayPacket.process(ClientPlayPacket.java:20)
	at Minestom Root ClassLoader//net.minestom.server.entity.Player.update(Player.java:328)
	at Minestom Root ClassLoader//net.minestom.server.entity.Entity.tick(Entity.java:662)
	at Minestom Root ClassLoader//net.minestom.server.thread.ThreadProvider.lambda$updateAndAwait$0(ThreadProvider.java:145)
	at java.base/java.util.concurrent.ConcurrentHashMap$KeySetView.forEach(ConcurrentHashMap.java:4706)
	at Minestom Root ClassLoader//net.minestom.server.thread.ThreadProvider.lambda$updateAndAwait$1(ThreadProvider.java:132)
	at Minestom Root ClassLoader//net.minestom.server.thread.TickThread$BatchRunnable.run(TickThread.java:64)
	at java.base/java.lang.Thread.run(Thread.java:831)
```
The index number varies based on the block broken but it's always negative.

**Comment by TheMode** [comment]
I believe that the issue has been fixed on the block branch #303

**Comment by TheMode** [comment]
Closing as this is fixed in master
